README.md

```diff
@@ -184,6 +184,23 @@ spec:
     # Array of strings (format is either with prefix '--key=value' or just 'value')
     # args: []
 
+    ## Audit Config
+    # https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy
+    # Define basic arguments for auditing log files. If needed, more complex flags
```

Comment:
would be nice to specify here mandatory fields, so users now which they need to specify.